clang/include/clang/Basic/Attr.td

```diff
@@ -4561,3 +4561,9 @@ def CodeAlign: StmtAttr {
     static constexpr int MaximumAlignment = 4096;
   }];
 }
+
+def ClspvLibclcBuiltin: DeclOrStmtAttr {
+  let Spellings = [Clang<"clspv_libclc_builtin">];
+  let Documentation = [Undocumented];
```

Comment:
Please add documentation anyway.  See `AttrDocs.td`

Reply:
What should be the documentation category?

Reply:
`DocCatFunction`?  As it seems to be a function decl attribute.  Subjects should be functions only it appears.